proto/frequenz/api/common/v1/pagination.proto

```diff
@@ -0,0 +1,34 @@
+// Parameters for pagination.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.pagination;
+
+// A message defining parameters for paginating list requests.
+message PaginationParams {
+  // [Optional]
+  // The maximum number of results to be returned per request.
+  // Defaults to 50 if omitted.
+  uint32 page_size = 1;
+
+  // [Optional]
+  // The token identifying a specific page of the list results.
+  string page_token = 2;
```

Comment:
please add `optional` as a prefix. I missed that.

Reply:
Super minor: what about `page_id`? I think it is a more common term.

Reply:
no sure if thats correct

https://protobuf.dev/programming-guides/api/#define-pagination-api

Reply:
OK, not in the protobuf world then :+1:

Reply:
Same. The word `Optional` in the docs looks incorrect since the field itself is not optional.

I think we want to make the `page_token` field optional? Should the default be to return the first page?

Reply:
> I think we want to make the page_token field optional? Should the default be to return the first page?

two times yes.